spec/models/chargeback_vm_spec.rb

```diff
@@ -1,3 +1,4 @@
+Chargeback
```

Comment:
@Fryguy @bdunne Please review this "load base class first" workaround so that we can use the following above:

```
class Chargeback < ActsAsArModel
  DescendantLoader.instance.load_subclasses(self)
```